backend/tracim_backend/tests/__init__.py

```diff
@@ -250,8 +259,14 @@ def override_settings(
         settings["sqlalchemy.url"] = "sqlite://"
         return settings
 
-    def init_database(self, settings: typing.Dict[str, typing.Any]) -> None:
-        self.engine = get_engine(settings)
+    def override_app_config(self) -> None:
```

Comment:
what is the usage of this method? need more docstring

Reply:
done.